pyout.py

```diff
@@ -43,19 +40,39 @@ class attribute.
     Attributes
     ----------
     term : blessings.Terminal instance
+
+    Examples
+    --------
+
+    Create a `Tabular` instance for two output fields, "name" and
+    "status".
+
+    >>> out = Tabular(["name", "status"], style={"status": {"width": 5}})
+
+    The first field, "name", is taken as the unique ID.  The `style`
+    argument is used to override the default width for the "status"
+    field that is defined by the class attribute `default_style`.
+
+    Write a row to stdout:
+
+    >>> out({"name": "foo", "status": "OK"})
+
+    Write another row, overriding the style:
+
+    >>> out({"name": "bar", "status": "BAD"},
+    ...     style={"status": {"color": "red", "bold": True}})
     """
 
     # TODO: Support things like auto-width, value-based coloring, etc.
     default_style = {"align": "<",
                      "width": 10,
                      "attrs": []}
```

Comment:
so is there still `attrs`?  I thought it is getting dissolved entirely from the level of API (i.e. inside attrs would be composed to be passed when dealing with Terminal backend)

Reply:
there is, based on the logic in 814baa2 message, but that's probably
unnecessary complication at this point.  I'll remove `attrs` from this
level.